### DISCUSSION TITLE
For "Lazer Tournament Client" add an option to turn off hotkeys

### DISCUSSION THREAD

**Discussion by Joryan59** [ppy/osu#27914] [Ideas] [answered]
i just noticed, on the lazer tourney client control panel you can type in the "team" to auto search the team name, but since hotkeys are always on (even while searching) it auto switches to that scene.
If there is a way that it can be turned off while typing in the control panel or a switch to turn it off that would be great

**Comment by Joehuu** [comment, marked as answer]
This is another case of https://github.com/ppy/osu/issues/25769. Textboxes should be blocking input when focused.

**Reply by frenzibyte** [reply]
This is essentially a duplicate of the issue thread above, and thus I'm marking this as answered.
